packages/integration/src/awsS3/config.ts

```diff
@@ -15,9 +15,8 @@
  */
 
 import { Config } from '@backstage/config';
-import { isValidHost } from '../helpers';
 
-const AMAZON_AWS_HOST = 'amazonaws.com';
+export const AMAZON_AWS_HOST = 'amazonaws.com';
```

Comment:
I think it would be nice to avoid this export tbh, feels like it's polluting a bit

Reply:
removed export